src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -6,8 +6,9 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <ServerGarbageCollection>false</ServerGarbageCollection>
     <InvariantGlobalization>true</InvariantGlobalization>
+    <LangVersion>preview</LangVersion>
     <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
-    <PublishAot>false</PublishAot>
+    <PublishAot>true</PublishAot>
```

Comment:
The crank benchmarks don't assume this is set to `false` do they?

Reply:
Ummm, good question. I know the force it on for the native aot run, but will need to check if they force it off for the normal runs.

Reply:
All good, it passes `/p:PublishAot=false` for the vanilla scenario

Reply:
https://github.com/aspnet/Benchmarks/blob/1e59fc99c0f9a98e37c2108b7a7d0077f2d90a2a/scenarios/goldilocks.benchmarks.yml#L141-L148